### DISCUSSION TITLE
CamelCase for swell-node

### DISCUSSION THREAD

**Discussion by y3owk1n** [swellstores/community#54] [Ideas]
By referring to the [front end doc](https://developers.swell.is/frontend-api/setup#usecamelcase), there is an option for getting the data in camel case. 

Whereas for swell-node library, is there a way to do the same as well? I couldn't find any relevant info [here](https://developers.swell.is/backend-api/backend-libraries/node#swell-node).

Thanks in advance!

**Comment by logeshswell** [comment]
Unfortunately, it's not implemented for `swell-node` at the moment.
But, still you could implement it by referring to the swell-js implementation [here](https://github.com/swellstores/swell-js/blob/fef31353fabce391cca56c31bb2a57b8ebf19560/src/utils/index.js#L63).

I am moving it to _Feature Request_ section to address it at the earliest.
